tests/unit/deferred.py

```diff
@@ -193,9 +196,26 @@ def test_apply_networking(switch, nic1, nic2, network, fresh_database):
                                             channel='vlan/native',
                                             type='modify_port')
 
+    action_native3 = model.NetworkingAction(nic=nic3,
+                                            new_network=None,
+                                            channel='',
+                                            type='revert_port')
```

Comment:
1. I'd suggest putting these in a list; variables like foo1 foo2 foo3 are generally a code smell.
2. This could use a comment pointing out that action_native3 is invalid, to trigger the failure below. Similarly, you should update the docstring to mention the additional thing this test is checking.